iocore/net/SSLConfig.cc

```diff
@@ -243,6 +245,21 @@ SSLConfigParams::initialize()
   ats_free(ssl_server_ca_cert_filename);
   ats_free(CACertRelativePath);
 
+  REC_ReadConfigStringAlloc(ticket_key_filename, "proxy.config.ssl.server.ticket_key.filename");
```

Comment:
The purpose of the newline is to break the code into "paragraphs" that make it easier to read. Getting a value from config is distinct from using that value for a particular purpose, so it deserves a new paragraph. Make it easy for the reader.